.circleci/config.pre.yml

```diff
@@ -72,6 +72,11 @@ commands:
               docker tag reachsh/$img:circleci-$CIRCLE_SHA1 reachsh/$img:$VERSION
             fi
           done
+  install_mo:
+    description: "Install mo"
+    steps:
+      - run: |
+          curl -sSL https://git.io/get-mo -o mo && chmod +x mo && sudo mv mo /usr/local/bin/
```

Comment:
I believe this is here because the examples are using mo; I think that code path should either NOT use mo OR the `one.sh` directory should install it if it isn't there using the correct `MO_URL`